dao/mongo.go

```diff
@@ -823,6 +803,13 @@ func (m *MongoService) GetResolutionResource(transactionID string) (models.Resol
 
 // DeleteResolutionResource deletes an resource filed for an Insolvency Case
 func (m *MongoService) DeleteResolutionResource(transactionID string) (int, error) {
+
+	httpStatus, err := m.DeleteResourceDBChecks(transactionID, "resolution")
+	return httpStatus, err
+
+}
+
+func (m *MongoService) DeleteResourceDBChecks(transactionID string, resType string) (int, error) {
```

Comment:
Is this the appropriate function name as it does delete the resource?

Reply:
Agree - I will change this.